Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java

```diff
@@ -101,6 +108,25 @@ public URI getUri() {
     // RestFUL API operations
     //
 
+    public Observable<Integer> getJobDegreeOfParallelism() {
+        return getJobList()
+                .flatMap(jobList -> Observable.from(jobList.value()))
```

Comment:
map() is enough?

Reply:
Firstly, the implementation of `values()`  method is as below.
```
    public List<JobInformationBasic> value() {
        return this.value;
    }
```

If we use `map` here, codes will look as below, which is strange to me to introduce java stream chain calls in rxjava chain calls.
```
        return getJobs()
                .map(jobList -> jobList.value().stream().mapToInt(jobInfo -> jobInfo.degreeOfParallelism()).sum())
                .defaultIfEmpty(0);
```
If we want to get rid of `stream`, we can replace `mapToInt()`  with `Lists.transform` , but I can't figure out a better way to sum up all integer values in java List without `stream`.

Therefore, I prefer the original `flatMap` way, which is much more clear and concise.